Week-17/book-list/src/Book/BookDetails.jsx

```diff
@@ -1,7 +1,8 @@
 import React, { useState } from 'react'
 import styles from "./stylesheet.module.css"
+import BookDataLoader from '../DataLoader/BookDataLoader'
 const BookDetails = (props) => {
-    const { val, i ,books,setBooks} = props
+    const { val, i, books, setBooks } = props
```

Comment:
please name vars so that it is meaningful, as of now I don't know what's val or i